src/Timer_Obscure_Behaviour.md

```diff
@@ -1,5 +1,29 @@
 # Timer obscure behaviour
 
+<!-- TODO: actually fix the diagrams -->
```

Comment:
This comment is to keep track of that TODO.

Hopefully the diagrams can be overhauled as part of this PR, but I'd rather have the band-aid and open an issue rather than hold this PR back, if that came to be.

Reply:
Is this done now?

Reply:
Almost all by b7f4844777eb73c8774fe95fbfcfd7a056e47f45; see https://github.com/gbdev/pandocs/pull/534#issuecomment-1897102969 for why the "almost".

tl;dr yes